src/lib.rs

```diff
@@ -12,6 +12,8 @@
 
 #![doc(html_root_url = "https://docs.rs/warpgrapher/0.6.0")]
 
+#[macro_use] extern crate maplit;
```

Comment:
If I remember right, I think macro_use is outdated syntax. You should be able to import macros from other crates using `use` statements without this kind of macro_use statement anymore.